argilla/src/argilla/_exceptions/_api.py

```diff
@@ -36,7 +36,7 @@ class BadRequestError(ArgillaAPIError):
 
 
 class ForbiddenError(ArgillaAPIError):
-    message = "Forbidden request to the server"
+    message = "User role is forbidden from performing this this action by server"
```

Comment:
What do you think about this message? The forbidden action could be raised not only when creating datasets but also when listing users (`client.users`) or getting datasets from workspaces without access (for admin users) 
```suggestion
    message = "Connected user has no privileges to perform this action on the server"
```